src/server/controllers/survey.controller.js

```diff
@@ -31,9 +33,20 @@ export function getRequest(req, res, next) {
         }
       })
       .then((surveyItem) => {
-        if (surveyItem)
+        if (surveyItem) {
           res.status(200).json(surveyItem);
-        else {
+        } else {
+          if (surveyId == 'A001-1') {
```

Comment:
1. Is this for mock survey? Or for the hard-coded survey?

2. Also, there should be more typed way to turn 'A001-1' into menteeData.surveyA001_1. Then you wouldn't need this if..else if... and just take care of it inline (turn string into enum type?)

3. Why is surveyId "A001-1" but enum name is "surveyA001_1"? (One is hypen "-", the other is underscore "_"). Please keep them consistent.